client/src/sass/main.sass

```diff
@@ -117,17 +117,18 @@
 
 .h5, h5
   color: $dk-text
-  font-size: 16px
+  font-size: 18px
```

Comment:
Here's hoping this project moves to relative units (ie, rem) to facilitate sizing/scaling in the near future.

https://www.google.com/search?q=why+use+rem+px

TLDR;
https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/

Reply:
Very cool, i like it. 
How does this work as per designs, where they are specified as specific px's. Is proper scaling between screen sizes just assumed and designs are based on the most standard screen sizes?

Reply:
It means we follow the designs but mostly convert to rems (except, say, small offset or the like).
12 px &rarr; 0.75rem
14 px &rarr; 0.875rem
16 px &rarr; 1rem
etc
And if possible, we use a few variations as possible.

Again, if you're in there, tweak it, otherwise it's future work.

Reply:
Definitely, this is relatively a quick adjustment in the base sass. Will do!

Reply:
What we _should_ do (in the other UIs as well), is make greater use of CSS variables, for sizes, etc.

Thanks for humouring me :)  Don't take on too much extra work atm.